linalg/ode.hpp

```diff
@@ -114,9 +118,9 @@ class ForwardEulerSolver : public ODESolver
    Vector dxdt;
 
 public:
-   virtual void Init(TimeDependentOperator &_f);
+   void Init(TimeDependentOperator &_f) override;
```

Comment:
Did you mean to remove the `virtual` keyword here?  In other words, is it the case that a class that derives the `ForwardEulerSolver` should no longer be able to redefine the `Init` function?

Reply:
Oops, I think I goofed here.  I believe that `Init` is still considered `virtual` even without the `virtual` keyword... I'm checking on that now.

Reply:
Yup, my mistake: `Init` is still considered `virtual`.  Please disregard my comment.